templates/docs/examples/patterns/navigation/deprecated-logo.html

```diff
@@ -148,10 +148,12 @@
   </div>
 </header>
 
-<script>
-  {% include 'docs/examples/patterns/navigation/_script-search.js' %}
-  {% include 'docs/examples/patterns/navigation/_script.js' %}
+{% block script %}
```

Comment:
Does this work? I'm not sure how nesting blocks like this would behave. I think they should be separate (script block should not be defined inside content block).